Adding Style Bundle & Updating Release Notes to Current

sunilarjun opened the pull request:
Updating the release notes with current release tied to https://github.com/rancher/elemental-docs/pull/465. Additionally updating the local playbook configuration and added the DSC style bundle in this PR as well.

btat reviewed (commented):
Any reason for using different UI bundles in the playbook files?

sunilarjun commented:
> Any reason for using different UI bundles in the playbook files?

Woops I blanked on updating the remote playbook so no reason, but added the DSC style bundle to that configuration in this commit: https://github.com/rancher/elemental-product-docs/pull/31/commits/711021a33beec32d32a84458ebb6fdff661c253c
Thanks for catching Billy!

btat commented:
> > Any reason for using different UI bundles in the playbook files?
> 
> Woops I blanked on updating the remote playbook so no reason, but added the DSC style bundle to that configuration in this commit: [711021a](https://github.com/rancher/elemental-product-docs/commit/711021a33beec32d32a84458ebb6fdff661c253c) Thanks for catching and the review Billy!

Gotcha thanks for updating Sunil. If we're fully swapping bundles in this PR, could you also remove/clean up remnants of the old bundle since it's no longer used.

btat reviewed (changes requested):
> > Any reason for using different UI bundles in the playbook files?
> 
> Woops I blanked on updating the remote playbook so no reason, but added the DSC style bundle to that configuration in this commit: [711021a](https://github.com/rancher/elemental-product-docs/commit/711021a33beec32d32a84458ebb6fdff661c253c) Thanks for catching and the review Billy!

Gotcha thanks for updating Sunil. If we're fully swapping bundles in this PR, could you also remove/clean up remnants of the old bundle since it's no longer used.

sunilarjun commented:
@btat I removed just an old file and removed the old style bundle from the gitmodules in this commit: https://github.com/rancher/elemental-product-docs/pull/31/commits/f56151ebcebf1820660c4e0616bfcd9a7c2e43a3
I think that's everything but if you could verify it would be appreciated, thanks Billy.
